docs/guide/ops/cli/cli-ref-guide.md

```diff
@@ -0,0 +1,310 @@
+---
+title: CLI Reference Guide
+layout: website-normal
+menu_parent: index.md
+children:
+- { section: List of Commands }
+- { section: Scopes }
+- { section: Abbreviations}
+- { section: Command Reference }
+- { section: Login}
+- { section: Applications}
+- { section: Entities}
+- { section: Sensors}
+- { section: Effectors}
+- { section: Policies}
+- { section: Activities}
+- { section: Miscellaneous}
+---
+
+## Usage
+{% highlight text %}
+NAME:
+   br - A Brooklyn command line client application
+
+USAGE:
+   br [global options] command [command options] [arguments...]
+{% endhighlight %}
+
+## List of Commands
+Commands whose description begins with a `*` character are particularly experimental
+and likely to change in upcoming releases.  
+
+{% highlight text %}
+COMMANDS:
+
+   access		Show access control
+   activity		Show the activity for an application / entity
+   add-catalog		* Add a new catalog item from the supplied YAML
+   add-children		* Add a child or children to this entity from the supplied YAML
+   application		Show the status and location of running applications
+   catalog		* List the available catalog applications
+   config		Show the config for an application or entity
+   delete		* Delete (expunge) a brooklyn application
+   deploy		Deploy a new application from the given YAML (read from file or stdin)
+   destroy-policy	Destroy a policy
+   effector		Show the effectors for an application or entity
+   entity		Show the entities of an application or entity
+   env			Show the ENV stream for a given activity
+   invoke		Invoke an effector of an application and entity
+   locations		* List the available locations
+   login		Login to brooklyn
+   policy		Show the policies for an application or entity
+   rename		Rename an application or entity
+   restart		Invoke restart effector on an application and entity
+   sensor		Show values of all sensors or named sensor for an application or entity
+   set			Set config for an entity
+   spec			Get the YAML spec used to create the entity, if available
+   start		Invoke start effector on an application and entity
+   start-policy		Start or resume a policy
+   stderr		Show the STDERR stream for a given activity
+   stdin		Show the STDIN stream for a given activity
+   stdout		Show the STDOUT stream for a given activity
+   stop			Invoke stop effector on an application and entity
+   stop-policy		Suspends a policy
+   tree			* Show the tree of all applications
+   version		Display the version of the connected Brooklyn
+   help			
+
+GLOBAL OPTIONS:
+   --help, -h		show help
+   --version, -v	print the version
```

Comment:
this is the version of the CLI client, right? If so, should probably be made explicit here.